src/jsp/jsp_cl.c

```diff
@@ -1328,6 +1329,8 @@ jsp_make_method_sig_list (PARSER_CONTEXT * parser, PT_NODE * node, method_sig_li
     DB_OBJECT *mop_p = jsp_find_stored_procedure (parsed_method_name);
     if (mop_p)
       {
+	AU_DISABLE (save);
```

Comment:
When mop is null, AU_ENABLE (save) is performed with the 'save' variable never initialized.

Reply:
Thank you!. I'm gonna move this line out of the if-statement.